src/SessionOne.java

```diff
@@ -1,35 +1,41 @@
 import java.util.Scanner;
 
 public class SessionOne {
+
     public static void main(String[] args) {
         Scanner input = new Scanner(System.in);
-        System.out.println("Press 1, if you want to see if it's an SessionOne number");
+        System.out.println("Press 1, if you want to see if it's an Armstrong number");
         System.out.println("Press 2, whether you want to see if it's an Strong number");
         System.out.println("Press 3, whether you want to see if it's an Perfect number");
 
         int option = input.nextInt();
+
         System.out.println("Enter a number");
         String number = String.valueOf(input.nextInt());
 
         switch (option) {
             case 1:
-                isAnAmstrongNumber(number);
+                isAnArmstrongNumber(number);
             case 2:
                 isAnStrongNumber(number);
```

Comment:
It is still the same, I understand that you might be tired, adjust your pace accordingly, otherwise this will become something you won't feel like doing anymore (And you know you will regret that, make this a pleasant thing, play music you like, eat a snack you really enjoy, drink a nice beverage, look at pics of Amsterdam during your pauses). Try to review the PR after you create it or update it.

```suggestion
                isAStrongNumber(number);
```

Reply:
Almost there, merge it's fine, you can see it below 👀

Reply:
oh no... I was sure that it was the correct way ... booo... sorry

Reply:
QQ, can I pull the code in feature/practice to another machine? for example Feature A in Caro's Local to Diana's locale? 😊 🙏

Reply:
Not exactly, this means there's something you haven't fully understood. Caro's feature branch is only in Caro's machine, nobody can touch it there except her. However Caro's feature branch is linked to a remote counterpart, if and only if Caro has already pushed it to the origin. If that's the case, Diana should be able to checkout the remote counterpart of Caro's feature branch on her own machine. There's no way to ensure that this remote counterpart is up to date with Caro's latest changes, Diana can only trust that Caro pushes often and pull whatever version of the branch is on the remote, it might be behind of Caro's local branch or even ahead if another contributor has pushed changes to this particular branch.

It may sounds like a disadvantage, but it's the greatest advantage of distributed version control systems (in my opinion) nobody messes with each others' work. Still, it leaves all the responsibility to keep the remote sources updated on the contributors. Keep that in mind when working, make meaningful commits and push them as often as you can to the remote, it's the best way to ensure that your work will be safe, which is one of the main reasons to use version control.

I am guessing you ask this because you want to have this on your job's laptop, so yeah, go ahead clone the project and checkout the branch, take a look at what she left there. (Having maven properly configured would make this easier, as it sets up dependencies and project's configuration, without relying on personal tools as the IDE, since Caro and Diana both use IntelliJ I think there won't be any issues)

Reply:
> Not exactly, this means there's something you haven't fully understood. Caro's feature branch is only in Caro's machine, nobody can touch it there except her. However Caro's feature branch is linked to a remote counterpart, if and only if Caro has already pushed it to the origin. If that's the case, Diana should be able to checkout the remote counterpart of Caro's feature branch on her own machine. There's no way to ensure that this remote counterpart is up to date with Caro's latest changes, Diana can only trust that Caro pushes often and pull whatever version of the branch is on the remote, it might be behind of Caro's local branch or even ahead if another contributor has pushed changes to this particular branch.
> 
> It may sounds like a disadvantage, but it's the greatest advantage of distributed version control systems (in my opinion) nobody messes with each others' work. Still, it leaves all the responsibility to keep the remote sources updated on the contributors. Keep that in mind when working, make meaningful commits and push them as often as you can to the remote, it's the best way to ensure that your work will be safe, which is one of the main reasons to use version control.
> 
> I am guessing you ask this because you want to have this on your job's laptop, so yeah, go ahead clone the project and checkout the branch, take a look at what she left there. (Having maven properly configured would make this easier, as it sets up dependencies and project's configuration, without relying on personal tools as the IDE, since Caro and Diana both use IntelliJ I think there won't be any issues).... Exactly, so.. guess what? I cloned this code in my job's laptop, but I didn't create any branch before committing the change, so I was like in a panic attack, ..., but now after reading this, it doesn't matter if I connect my local master to the right feature/practice in the remote side, right? please say yes 🙏

Reply:
If what you are saying here is that you linked your `master` branch to `origin/feature/practice` (which I hope it's not the case) something like that shouldn't be done. Local branches should only be linked to their remote counterparts, `master > origin/master`, `feature/practice  > origin/feature/practice`. Even when you can link any local branch with any remote one, it doesn't mean that you should do it, will only leads to confusion and it goes against the pattern. So if you are in your local master branch right now and you run `git status` the message you should be seeing is 
```
On branch master
Your branch is up-to-date with 'origin/master'.
```
Same applies for a feature branch:

```
On branch feature/practice
Your branch is up-to-date with 'origin/feature/practice'.
```
Of course the branches should be up to date, if not the message will differ a bit, but you will be able to tell which branch are you currently pointing out in the remote. So, let's make this happen, run the following:

```
// this command will update your local with the latest changes, new branches,
// commits etc, so you will have access to these resources from your local repo
git fetch

// this command will show you all your local and remote branches, read carefully
git branch -a

// if the branch exists as a remote one, but not at local level, this command
// will create a new local branch, will link it to the remote counterpart,
// and will pull whatever is on the remote branch as well
git checkout feature/practice
```
Now, to fix the wrong branches link mess up (Probably is better to do this after you merge the pull request):

```
git checkout master
git push --set-upstream origin master
```
> When you do this, it's most likely that there will be conflicts, hopefully there won't be, if there are, you will have to pull, reconcile the histories between branches, solve conflicts if there's any, and push again to the remote counterpart. Give it a try, get messy, everything is fixable.